<!-- FILEPATH: /home/abivishaq/grasp_test_ws/src/rerail_stretchit_grasp/stretch_fetch_grasp_bridge/launch/grasp_suggestor.launch -->

<launch>

  <!-- Launch grasp_rosbag.launch -->
  <!-- <include file="$(find grasp_pkg_tests)/launch/grasp_rosbag.launch"/> -->
  <!-- roslaunch stretch_core d435i_low_resolution.launch -->
  <!-- <include file="$(find stretch_core)/launch/d435i_low_resolution.launch"/> -->

  <!-- Launch grasp_suggestion_testing.launch with cloud_topic argument -->
  <include file="$(find fetch_grasp_suggestion)/launch/grasp_suggestion_testing.launch">
    <arg name="cloud_topic" value="/camera/depth/color/points"/>
  </include>

  <!-- Launch grasp_suggestion_testing_frontend.launch -->
  <include file="$(find fetch_grasp_suggestion)/launch/grasp_suggestion_testing_frontend.launch"/>

  <!-- Run stretch_grasp_filter.py -->
  <node name="stretch_grasp_filter" pkg="stretch_fetch_grasp_bridge" type="stretch_grasp_filter.py"/>
  <node name="stretch_grasp_suggestor" pkg="stretch_fetch_grasp_bridge" type="stretch_grasp_suggestion.py"/>


</launch>
